common-service-core/src/main/java/org/zowe/apiml/zaas/ZaasSafIdtResponse.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.zaas;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ZaasSafIdtResponse {
```

Comment:
I would rather use ZaasTokenResponse instead. It will make processing in CGW easier. Just leave the cookie empty.